pkgs/development/node-packages/node-packages-v12.nix

```diff
@@ -13,13 +13,13 @@ let
         sha512 = "nne9/IiQ/hzIhY6pdDnbBtz7DjPTKrY00P/zvPSm5pOFkl6xuGrGnXn/VtTNNfNtAfZ9/1RtehkszU9qcTii0Q==";
       };
     };
-    "ajv-6.10.2" = {
```

Comment:
We probably don't need to commit changes to the v12 or v13 package sets when you're just trying to update the v10 one.